docs/installing.md

```diff
@@ -97,29 +101,42 @@ To run PrairieLearn locally with external grader and workspace support, create a
 mkdir "$HOME/pl_ag_jobs"
 ```
 
-Now, run PrairieLearn as usual, but with additional options. For example, if your course directory is in `$HOME/pl-tam212` and the jobs directory created above is in `$HOME/pl_ag_jobs`, and you are using Linux or Mac OS X, the new command is as follows:
+#### Platform-specific invocations
```

Comment:
Could we get away with combining these into one command for all platforms? Adding `--platform linux/amd64` unconditionally should be fine, and in my testing, `--add-host=host.docker.internal:172.17.0.1` didn't break anything on macOS.

Reply:
OK, I've tried combining the platform invocations into a single use case (latest commits).